argo/workflow/dm.yaml

```diff
@@ -0,0 +1,96 @@
+metadata:
+  generateName: tipocket-dm-
+  namespace: argo
+spec:
+  entrypoint: call-tipocket-dm
+  arguments:
+    parameters:
+      - name: ns
+        value: tipocket-dm
+      - name: hub
+        value: "docker.io"
+      - name: repository
+        value: "pingcap"
+      - name: image_version
+        value: nightly
+      - name: tidb_image
```

Comment:
ditto.

Reply:
renamed in https://github.com/pingcap/tipocket/pull/220/commits/44941bc7c203f226fba4676a32197d038630c4b8.